mastodon/table_mastodon_toot_list.go

```diff
@@ -0,0 +1,84 @@
+package mastodon
+
+import (
+	"context"
+
+	"github.com/mattn/go-mastodon"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+)
+
+func tableMastodonTootList() *plugin.Table {
+	return &plugin.Table{
+		Name:        "mastodon_toot_list",
+		Description: "Represents a toot on your list timeline.",
+		List: &plugin.ListConfig{
```

Comment:
Should this be represented by a `GetConfig` instead of `ListConfig` since we have a required qual?

Reply:
No, in this case list function is listing toots of your Mastodon account list.
It's confusing, I know